semantic_release/cli.py

```diff
@@ -292,9 +292,10 @@ def publish(**kwargs):
         if upload_release and check_token():
             logger.info("Uploading to HVCS release")
             upload_to_release(owner, name, new_version, dist_path)
-        # Remove distribution files as they are no longer needed
-        if remove_dist:
-            remove_dists(dist_path)
+
+            # Remove distribution files as they are no longer needed
+            if remove_dist:
```

Comment:
Instead of moving this in here change it to `if (upload_pypi or upload_release)  and remove_dist:`

Reply:
👍 
updated